sdk/tests/deqp/functional/gles3/es3fFboTestCase.js

```diff
@@ -135,7 +135,7 @@ var DE_ASSERT = function(x) {
             /** @const @type {goog.NumberArray} */ var supportedSampleCounts = es3fFboTestCase.querySampleCounts(sizedFormat);
             var supported = Array.prototype.slice.call(supportedSampleCounts);
             if (supported.indexOf(numSamples) == -1) {
-                if (numSamples >= 2) {
+                if (minSampleCount == 0 || numSamples > gl.getParameter(gl.MAX_SAMPLES)) {
```

Comment:
The c++ code throws a NotSupportedError if the numSamples isn't supported.  Is that a test failure?  I thought it's actually not a failure.  In that case, we don't really need to do this if-else here, we can just do "testPassed('Sample count not supported'); return false;"